README.md

```diff
@@ -226,6 +226,14 @@ class AlbumAdmin(admin.ModelAdmin):
         """As above..."""
 ```
 
+Contributing:
+------------
+This project is a combined effort of a lot of selfless developers who try to make things easier. Your contribution is most welcome.
+
+Please make a PR to the branch `pre_release` and make sure your branch does not have any conflicts and clearly mention the problems or improvements your PR is addressing.
```

Comment:
Wonder if this could be automated by changing the repo default branch setting!

Reply:
Ah! good point man, did that! Thanks.